### DISCUSSION TITLE
Docked window does not retain size and location when OBS loads

### DISCUSSION THREAD

**Discussion by scott-korin** [sorayuki/obs-multi-rtmp#91] [General]
If I adjust the size and location of the "multiple output" window within my docked area of OBS, when I close OBS and then open it again, **_all of my changes to my docked views are reset._** (including other windows in the dock). This only happens when included the "multiple output" window in my docked area.

**Comment by bucefalo2** [comment]
same happens to me

**Comment by sorayuki** [comment]
hi, it's fixed in 0.2.7

**Reply by bucefalo2** [reply]
beautiful, thank you! beer on the way 💯🍺
PayPal asking crazy fee, I try later with wechat 🍾🍻

**Reply by bucefalo2** [reply]
ok.. done now on wechat !! thank you !!!!!!!!!

**Reply by sorayuki** [reply]
> beautiful, thank you! beer on the way 💯🍺
> PayPal asking crazy fee, I try later with wechat 🍾🍻

Yes, it does ask crazy fee lol

**Reply by iskconjaipur** [reply]
No it is not fixed.

and even the plugin dock on the OBS isn't very clear, what version it is running.
